Eatery Blue/UI/EateryScreen/EateryModelController.swift

```diff
@@ -233,6 +233,10 @@ class EateryModelController: EateryViewController {
             if menuCategory.category == "Chef's Table - Sides" {
                 sortedCategories.swapAt(1, i)
             }
+//            if menuCategory.category == "Grill" {
```

Comment:
Probably delete this commented out code + print statement, but otherwise everything looks great!